gradle.properties

```
@@ -7,7 +7,7 @@ cordaGradlePluginsVersion=5.0.11
 cordaPlatformVersion=8
 cordaReleaseGroup=net.corda
 cordaUpdatesVersion=2.0
-cordaVersion=4.6-SNAPSHOT
+cordaVersion=4.7-HC01
```

Maybe now we should put:
```suggestion
cordaVersion=4.7-RC01
```